src/utils/node/dynamicProps.js

```diff
@@ -52,20 +74,31 @@ async function modifyComponentTypes(filePath, props) {
     const targetFile = path.join(filePath, "types.ts");
 
     /* Read `types.ts` and split it by lines. */
-    const content = await fs.readFile(targetFile, "utf-8");
-    const lines = content.split("\n");
+    let fileContent = await fs.readFile(targetFile, "utf-8");
+    const propsInterface = fileContent.match(propsInterfaceRegExp)?.at(0)?.trim();
+
+    // Remove props interface and double returns from fileContent
```

Comment:
use /* ... */ comments here.

`double returns from fileContent` - what you mean?

Reply:
not fixed